packages/react-magma-dom/src/components/TreeView/useTreeView.ts

```diff
@@ -377,7 +379,27 @@ export function useTreeView(props: UseTreeViewProps) {
               isDisabled,
             })
           );
-        }
+        },
+
+        expandAll() {
+          const expandableIds = items
+            .filter(item => item.hasOwnTreeItems)
+            .map(item => item.itemId);
+
+          setExpandedSet(new Set(expandableIds));
+
+          if (onExpandedChange) {
+            onExpandedChange({} as React.SyntheticEvent, expandableIds);
+          }
```

Comment:
We call this in other places like:
```
     onExpandedChange &&
        typeof onExpandedChange === 'function' &&
        onExpandedChange({} as React.SyntheticEvent, expandableIds)
```

See `handleExpandedChange` in useTreeView

(same comment for collapseAll)